sphinx_scylladb_theme/side-nav.html

```diff
@@ -1,4 +1,4 @@
-<div id="side-nav" class="side-nav custom-scroll-bar" data-closable data-toggler=".show">
+<div id="side-nav" class="side-nav @apply dark:bg-navigation-base custom-scroll-bar" data-closable data-toggler=".show">
```

Comment:
```suggestion
<div id="side-nav" class="side-nav dark:bg-navigation-base custom-scroll-bar" data-closable data-toggler=".show">
```

I don't think you need `@apply` here since you're already on a `"class"`